apollo-api/src/commonMain/kotlin/com/apollographql/apollo/api/CustomTypeValue.kt

```diff
@@ -13,11 +16,12 @@ sealed class CustomTypeValue<T>(@JvmField val value: T) {
     override fun equals(other: Any?): Boolean {
       if (this === other) return true
       if (other !is GraphQLString) return false
+      if (value != other.value) return false
       return true
     }
 
     override fun hashCode(): Int {
-      return javaClass.hashCode()
```

Comment:
Here `hashCode` has been always the same value. Was that intentional?

Reply:
Probably oversight